### ISSUE TITLE
[Large Screen] Keyboard navigation skips or is unable to select and click on certain UIs

### ISSUE THREAD

**Issue by samartnik** [brave/brave-browser#52099]
### Description

Reported by Large Screen Audit:
Keyboard navigation skips or is unable to select and click on certain UIs. The items below are solely from the app’s UI, though the web page is tested as well:
- Main page
  - Top row of icons and tabs
    - Only able to select the options icon and the address/search bar field (not reproducible, I was able cycle through all the icons and address bar using arrows)
    - Not able to select any of the tabs (the same in the upstream)
- History
  - Cannot enter the list of pages (not reproducible, I was able to enter the list of pages using Tab)

### Steps to reproduce

See description

### Actual result

n/a

### Expected result

n/a

### Reproduces how often

Easily reproduced

### Brave version

n/a

### Device

- Brand/model: n/a
- Android version: n/a

### Channel information

- [ ] release (stable)
- [ ] beta
- [ ] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

**Comment by samartnik** [comment]
Closing as it's either not reproducible anymore or has the same behavior in the upstream
